resources/concise-guide-to-fixing-web-standards.md

```diff
@@ -0,0 +1,51 @@
+# Concise Guide to Identifying (and resolving) a Web Platform Bug
+
+Web developers rely on a great many software packages, libraries, and tools to build applications. End users expect to have the same experience on these applications regardless of their environment. The web is an ever-evolving platform to target as platforms add new features and APIs, and web standards bodies incorporate them into final specifications. Thus, hitting a bug can be a frustrating experience particularly when it is not clear _where_ the bug originates - is it in your code, the library code, the browser, the specification, or something else? Below are some questions to guide you through determining whether you have found a platform bug, and how to report it.
+
+
+
+1. **Reduce, reduce, reduce**
+
+    Bugs often appear in complex setups, and it can be unclear which part or which feature the bug is stemming from. The first step of any bug investigation is to eliminate code unrelated to the bug so that its causes can be more clearly investigated. This is called a _reduced testcase_ or _reduction_. A reduction will not only help you understand what is happening much better, it will also help you communicate about the issue with spec developers, browser engineers, and other stakeholders. Moreover, it will also help your bug being fixed faster: bugs that include links to reduced testcases are generally triaged a lot faster.  \
+ \
+Copy over the current page, script, or even your entire project and start aggressively *removing* code, checking that you can still reproduce the bug with the code left. This also applies to the UI: Shorten any non-relevant text, eliminate any unrelated CSS styling, simplify any HTML subtree. You can almost always reduce more than you think. Most bugs can be reduced to &lt; 20 lines of code, often significantly less than that. If you’ve used unit tests before, it's a very similar process: imagine that you are writing a unit test to test if that particular bug is fixed.  
+
+
+
+    These additional resources may be helpful in learning how to create better reduced testcases: 
+	* [https://webkit.org/test-case-reduction/](https://webkit.org/test-case-reduction/)
+   * [https://css-tricks.com/reduced-test-cases/](https://css-tricks.com/reduced-test-cases/) 
+
+
+2. **What kind of bug do you have?** \
```

Comment:
```suggestion
2. **What kind of bug do you have?**
```
i think all these backslashes at the end of lines need to be removed? (and trailing whitespace as a result)

Reply:
weird, that must've come from the docs-to-markdown generator tool....